src/App/Components/Comment/comment.icons.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
check why you are disabling these es-lint rules?

Reply:
@MCFrank16 why??